src/index.njk

```diff
@@ -61,39 +61,82 @@ hubspot:
         </div>
     </div>
 </div>
-<div class="about w-full bg-gray-50 pt-16">
-    <!-- FlowFuse Benefits -->
-    <div class="w-full px-6 pt-2">
+<div class="about w-full bg-gray-50 pt-16 px-6">
+    <!-- Industrial Automation -->
+    <div class="w-full pt-2">
         <div class="max-w-md sm:max-w-screen-lg mx-auto">
-            <div class="mx-auto w-full md:max-w-screen-lg grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-x-8 gap-y-14 pb-14">
-                <h2 class="text-center w-full md:text-left col-span-full">FlowFuse Creates <span class="text-indigo-600">Value</span></h2>
-                {% for item in home %}
-                <div class="relative w-full">
-                    <div class="flex flex-col items-center sm:items-start">
-                        <div class="flex flex-col justify-center md:justify-start gap-3 w-full">
-                            <div class="w-8 h-8 m-auto sm:m-0 text-indigo-600">
-                                {% set svgPath = "components/icons/" + item.icon + ".svg" %}
-                                {% include svgPath %}
-                            </div>
-                            <div class="w-full flex flex-row gap-3 mx-auto md:m-0">
-                                <h5 class="w-full md:m-0">
-                                    <div class="text-xl text-gray-600 text-center sm:text-left">{{ item.title | safe }}
+            <div class="mx-auto w-full md:max-w-screen-lg gap-x-8 gap-y-14 pb-14">
+                <h2 class="text-center w-full md:text-left">End-to-end <span class="text-indigo-600">industrial automation</span></h2>
```

Comment:
```suggestion
                <h2 class="text-center w-full md:text-left">End-to-end <span class="text-indigo-600">Industrial Data Platform</span></h2>
```

Reply:
but this would narrow it down to data, wouldn't it? what about the interactive part? you can interact with more than just data, right?